metricbeat/module/http/json/json.go

```diff
@@ -153,3 +162,16 @@ func (m *MetricSet) getHeaders(header http.Header) map[string]string {
 	}
 	return headers
 }
+
+func replaceDots(data interface{}) interface{} {
```

Comment:
I wonder if we should have this recursive function also in libbeat later on.

Reply:
I actually had the very same idea as it is quite likely that also beats have json as payload where in all keys "dots" shall be replaced.

I can open a new PR for this if you OK.

Reply:
Sounds like a good idea.